### DISCUSSION TITLE
complete the psql command with host parameter

### DISCUSSION THREAD

**Discussion by bnicu** [stefanzweifel/laravel-backup-restore#44] [Ideas]
Can you please add the -h parameter for the host? ex in src/Databases/PostgreSql.php  27
return 'psql -h '.config('database.connections.pgsql.host').' -U '.config('database.connections.pgsql.username').' -d '.config('database.connections.pgsql.database').' < '.$dumpFile;

**Comment by stefanzweifel** [comment]
Sure thing @bnicu, will add this in the next few days.
If you want to, you can also contribute this on your own.

You can edit the respective file directly here on GitHub and add the `-h` option yourself:
https://github.com/stefanzweifel/laravel-backup-restore/blob/main/src/Databases/PostgreSql.php#L26-L41

**Reply by stefanzweifel** [reply]
Thanks for your PR @bnicu.
For now, I've solved this `--host`-option issue differently in #46 and created a new release: https://github.com/stefanzweifel/laravel-backup-restore/releases/tag/v1.1.4.

Will add proper support for database urls for all supported databases in a future release.
